work/src/org/kuali/kfs/module/ar/report/service/impl/ContractsGrantsInvoiceReportServiceImpl.java

```diff
@@ -63,39 +63,9 @@
 import org.kuali.kfs.sys.service.NonTransactional;
 import org.kuali.kfs.sys.service.OptionsService;
 import org.kuali.kfs.sys.service.ReportGenerationService;
-import org.kuali.rice.core.api.config.property.ConfigurationService;
```

Comment:
No, I think eclipse freaked out on me and killed some imports. I need to get them back in.